compiler/rustc_target/src/asm/mod.rs

```diff
@@ -1278,6 +1283,14 @@ impl InlineAsmClobberAbi {
                     a8, a9, a10, a11, a12, a13, a14, a15,
                 }
             },
+            InlineAsmClobberAbi::Bpf => clobbered_regs! {
+                Bpf BpfInlineAsmReg {
+                    // Refs: Section 1.1 "Registers and calling convention" in BPF ABI Recommended Conventions and Guidelines v1.0
+                    // https://github.com/torvalds/linux/blob/ffd294d346d185b70e28b1a28abe367bbfe53c04/Documentation/bpf/standardization/abi.rst#id2
```

Comment:
Perhaps even better if we link to the html docs?

https://www.kernel.org/doc./html/latest/bpf/standardization/abi.html

Reply:
I used a link to a specific version of the source code because of my previous experience of using a link to https://www.kernel.org/doc./html/latest/ elsewhere and having trouble when the page being renamed.

Reply:
Well, we use links to https://www.kernel.org/doc in some other places, so I switched to using it for this PR as well.

https://github.com/search?q=repo%3Arust-lang%2Frust%20%2F%5C%2F%5C%2Fwww%5C.kernel%5C.org%5C%2Fdoc%2F&type=code

Reply:
Link to the anchor? https://www.kernel.org/doc/html/latest/bpf/standardization/abi.html#registers-and-calling-convention